xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -873,6 +877,8 @@ unsigned int CAESinkAUDIOTRACK::AddPackets(uint8_t **data, unsigned int frames,
       double time_off = time_should_ms - time_to_add_ms;
       if (time_off > 0)
         usleep(time_off * 500); // sleep half the error away
+
+      CLog::Log(LOGINFO, "m_delay %lf ms period_time %lf ms Adding-Time %lf ms Sleeping-Time %lf ms", m_delay * 1000, period_time * 1000, time_should_ms, time_off / 2);
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)